package.json

```diff
@@ -7,7 +7,7 @@
     "npm": ">=7.24.2"
   },
   "dependencies": {
-    "alephium-js": "1.1.0",
+    "alephium-js": "file:../alephium-js",
```

Comment:
This should be changed before merging. Also, using `npm link` removes the need to change this.

Reply:
It will need to be updated no matter what because it relies on api-explorer.ts updates - I'll leave this comment open until that happens